db/fixtures/miq_product_features.yml

```diff
@@ -1493,6 +1497,11 @@
       :description: Modify Saved Reports
       :feature_type: admin
       :identifier: miq_report_saved_reports_admin
+    - :name: Reload
+      :description: Reload Saved Reports
+      :feature_type: view
```

Comment:
So weird that this needed a dedicated feature - I would think if you have miq_report_saved_reports_view, then you get reload altuomatically.